packages/esm-patient-chart-app/src/banner/contact-details/contact-details.component.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { InlineLoading } from '@carbon/react';
+import { useRelationships } from './relationships.resource';
+import { usePatientContactAttributes } from '../hooks/usePatientAttributes';
+import styles from './contact-details.scss';
+
+interface ContactDetailsProps {
+  address: Array<fhir.Address>;
+  telecom: Array<fhir.ContactPoint>;
+  patientId: string;
+}
+
+const Address: React.FC<{ address?: fhir.Address }> = ({ address }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <p className={styles.heading}>{t('address', 'Address')}</p>
+      <ul>
+        {address ? (
+          <>
+            <li>{address.postalCode}</li>
+            <li>{address.city}</li>
+            <li>{address.state}</li>
+            <li>{address.country}</li>
+          </>
+        ) : (
+          '--'
+        )}
+      </ul>
+    </>
+  );
+};
+
+const Contact: React.FC<{ telecom: Array<fhir.ContactPoint>; patientUuid: string }> = ({ telecom, patientUuid }) => {
+  const { t } = useTranslation();
+  const value = telecom?.length ? telecom[0].value : '--';
+  const { isLoading, contactAttributes } = usePatientContactAttributes(patientUuid);
+
+  return (
+    <>
+      <p className={styles.heading}>{t('contactDetails', 'Contact Details')}</p>
+      <ul>
+        <li>{value}</li>
+        {isLoading ? (
+          <InlineLoading description={t('loading', 'Loading...')} />
```

Comment:
Please use a TextSkeleton for loading purposes. According to the designs, skeletons are to be shown when the data is loading.
Thanks!

Reply:
@vasharma05 what is a TextSkeleton?